src/main/java/io/github/codenilson/lavava2025/entities/Player.java

```diff
@@ -36,6 +39,10 @@ public class Player {
 
     private String agent;
 
+    @ElementCollection(fetch = FetchType.EAGER)
+    @CollectionTable
```

Comment:
É melhor usar uma entidade `role` e substituir isso. Ou usar um enum para evitar erros de digitação.

```suggestion
    @OneToMany(mappedBy = "player")
    private Set<PlayerRole> roles = new HashSet<>();
```